tests/api.c

```diff
@@ -7841,8 +7843,8 @@ static int test_wolfSSL_CTX_verifyDepth_ServerClient_2_ctx_ready(
     WOLFSSL_CTX* ctx)
 {
     wolfSSL_CTX_set_verify(ctx, SSL_VERIFY_PEER, myVerify);
-    myVerifyAction = VERIFY_USE_PREVERFIY;
```

Comment:
Is this test maintained in another location or was it never correct to have the verify action be VERIFY_USE_PREVERIFY

Reply:
From what I can tell, it was incorrectly ported to the `Expect` test API. I'm not sure why its passing without my PR. Maybe the full chain wasn't being checked somehow.